tests/unit/providers/vector_io/test_sqlite_vec.py

```diff
@@ -82,14 +82,41 @@ async def test_add_chunks(sqlite_vec_index, sample_chunks, sample_embeddings):
 
 
 @pytest.mark.asyncio
-async def test_query_chunks(sqlite_vec_index, sample_chunks, sample_embeddings):
+async def test_query_chunks_vector(sqlite_vec_index, sample_chunks, sample_embeddings):
     await sqlite_vec_index.add_chunks(sample_chunks, sample_embeddings)
+
     query_embedding = np.random.rand(EMBEDDING_DIMENSION).astype(np.float32)
-    response = await sqlite_vec_index.query(query_embedding, k=2, score_threshold=0.0)
+    response = await sqlite_vec_index.query(
+        query_embedding, query_str="", k=2, score_threshold=0.0, search_mode="vector"
+    )
     assert isinstance(response, QueryChunksResponse)
     assert len(response.chunks) == 2
 
 
+import pytest
+
+
+@pytest.mark.asyncio
+async def test_query_chunks_full_text_search(sqlite_vec_index, sample_chunks, sample_embeddings):
```

Comment:
I would add a test to include where `k>j` for some `j>0` where j is the returned number of chunks.